packages/main/src/SplitButton.js

```diff
@@ -0,0 +1,449 @@
+		 *
+		 * @type {Object}
+		 * @private
+		 */
+		_splitButtonAccInfo: {
+			type: Object,
+		},
+
+		/**
+		 * Sets the accessible aria name of the component.
+		 *
+		 * @type {String}
+		 * @defaultvalue: ""
+		 * @public
+		 */
+		 accessibleName: {
+			type: String,
+			defaultValue: undefined,
+		},
+
+		/**
+		 * Defines the tabIndex of the component.
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		_tabIndex: {
+			type: String,
+			defaultValue: "0",
+			noAttribute: true,
+		},
+
+		/**
+		 * Indicates if an update of the tabIndex of the component must be performed
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_tabIndexUpdate: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Indicates if there is action on default or arrow button triggered by keyboard
+		 * (Space, Enter, F4, Arrow Up/Down, Alt+Arrow Up/Down)
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_keyboardAction: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Indicates if there is SHIFT or ESCAPE key pressed
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		_shiftOrEscapePressed: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the icon of the text button
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @private
+		 */
+		 _textButtonIcon: {
+			type: String,
+			defaultValue: "",
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the active state of the text button
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		 _textButtonActive: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+		 * Defines the active state of the arrow button
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
+		 */
+		 _arrowButtonActive: {
+			type: Boolean,
+			noAttribute: true,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.SplitButton.prototype */ {
+		/**
+		 * Fired when the user clicks on the default action.
+		 */
+		"click": {},
+
+		/**
+		 * Fired when the user clicks on the arrow action.
+		 */
+		"arrow-click": {},
+	 },
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * <code>ui5-split-button</code> enables users to trigger actions. It is constructed of two separate actions -
+ * default action and arrow action that can be activated by clicking or tapping, or by
+ * pressing certain keyboard keys - <code>Space</code> or <code>Enter</code> for default action,
+ * and <code>Arrow Down</code> or <code>Arrow Up</code> for arrow action.
+ *
+ * <h3>Usage</h3>
+ *
+ * <code>ui5-split-button</code> consists two separate buttons:
+ * <ul>
+ * <li>for the first one (default action) you can define some <code>text</code> or an <code>icon</code>, or both.
+ * Also, it is possible to define different icon for active state of this button - <code>activeIcon</code>.</li>
+ * <li>the second one (arrow action) consists only <code>slim-arrow-down</code> icon.</li>
```

Comment:
Perhaps it should be consists '**of**', but I suggest getting a review by a KM for all the docs.

Reply:
Done.
Our KM will be assigned for review too.